content/en/aws/lambda/index.md

```diff
@@ -10,14 +10,72 @@ AWS Lambda is a Serverless Function as a Service (FaaS) system that allows you t
 
 LocalStack allows you to execute your Lambda functions locally, without the need to deploy them to AWS. This is a great way to test your code, and to learn more about how your Lambda functions work, before deploying them to AWS. LocalStack allows you to execute your Lambda functions, in various execution modes, which is detailed on our [Lambda execution modes]({{< ref "lambda-executors" >}}) page.
 
+## Special tooling for Lambdas
+
 We also provide tools to help you develop, debug and test your Lambda functions more efficiently:
 
 - [Hot swapping]({{< ref "hot-swapping" >}}): Hot code swapping for Lambda functions using LocalStack’s code mounting feature.
 - [Remote debugging]({{< ref "debugging" >}}): Attaching a debugger to your Lambda function using your IDE.
 
-LocalStack Pro samples contain significant code examples that demonstrate how to use LocalStack to execute Lambda functions:
+## Lambda sample applications
+
+LocalStack Pro samples contains a number of code examples that demonstrate how to use LocalStack to execute Lambda functions:
 
 - [Lambda XRay Tracing](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-xray): Simple demo application illustrating Lambda XRay tracing using LocalStack, deployed via the Serverless framework.
 - [Lambda Container Images](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-container-image): Simple demo application illustrating Lambda container images in LocalStack. The Lambda image is built using Docker and pushed to a local ECR registry.
 - [Lambda Code Mounting and Debugging](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-mounting-and-debugging): Simple demo application to illustrate debugging Lambdas locally.
 - [Lambda Layers](https://github.com/localstack/localstack-pro-samples/blob/master/serverless-lambda-layers): Simple demo application illustrating Lambda layers using LocalStack, deployed via the Serverless framework.
+
+## Lambda Layers
+
+[Lambda layers](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html) are an AWS feature that allows to pull in additional code and content into your Lambda functions.
+
+LocalStack Pro provides support for deploying Lambda layers locally - for more details on general usage, please follow the documentation and examples on the [AWS website](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html).
```

Comment:
```suggestion
LocalStack Pro provides support for deploying Lambda layers locally - for more details on general usage, please follow the documentation and examples in the [AWS documentation on Lambda layers](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html).
```
might be personal preference, but I think for accessibility it is better to label the link in a way where it would be understandable without the surrounding context. Not sure if this is really necessary here, but 'AWS website' does not seem like a clear description about whats behind the link.

Reply:
Great point, thanks! 👍